src/main/java/org/gridsuite/boundary/importer/job/utils/CgmesBoundaryUtils.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.boundary.importer.job.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+public final class CgmesBoundaryUtils {
+    public static final int MAX_ZIP_ENTRIES_COUNT = 100;
+    public static final int MAX_ZIP_SIZE = 1000000000;
+    public static final String TPBD_FILE_REGEX = "^(.*?(__ENTSOE_TPBD_).*(.xml))$";
+    public static final String EQBD_FILE_REGEX = "^(.*?(__ENTSOE_EQBD_).*(.xml))$";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CgmesBoundaryUtils.class);
+
+    private CgmesBoundaryUtils() {
+    }
+
+    /* The boundary zip container file name should have the following structure:
+    <effectiveDateTime>__ENTSOE_BD_<fileVersion>.zip
+    where :
+    <effectiveDateTime>: UTC datetime (YYYYMMDDTHHmmZ)
+    <fileVersion>: three characters long positive integer number between 000 and 999
+     */
+    private static final String DOT_REGEX = "\\.";
```

Comment:
I prerfer [.] for the dot regex, as it doesn't risks multiple evaluations

Reply:
Done